WordPress/Classes/ViewRelated/NUX/SiteCreationCategoryTableViewController.swift

```diff
@@ -21,6 +21,7 @@ class SiteCreationCategoryTableViewController: NUXTableViewController {
         super.viewWillAppear(animated)
         setupTable()
         setupNavBar()
+        WPAppAnalytics.track(.createSiteProcessBegun)
```

Comment:
One last nitpick.  If we bump the stat here it could potentially be bumped multiple times if a user explored the different options available on the screen: blog, site, profile.  Maybe we could bump it only once when the controller is created?